### PR TITLE
Changed subscription and invoice statuses

### PR DESCRIPTION
**Pull request by adamaltman**
- added Subscription billingStatus

### CODE REVIEW

File: spec/components/schemas/Subscription.yaml [hunk 1]
```diff
@@ -22,13 +22,34 @@ properties:
...
+      - suspended
       - paused
+  billingStatus:
+    description: |
+      The billing status of the most recent invoice.  It may
+      help you determine if you should change the service status
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
```
Comment (slavcodev) [inline spec/components/schemas/Subscription.yaml]: Maybe would be nice to segregate it into new definition in order to re-use.

Comment (vova07) [reply spec/components/schemas/Subscription.yaml]: Yes, so we should not update 2 places in case it will change in the future.

Comment (2e3s) [reply spec/components/schemas/Subscription.yaml]: It's probably true, probably not worth it: not all invoice statuses may be supported here like `draft`

Comment (adamaltman) [reply spec/components/schemas/Subscription.yaml]: That's why I didn't do that (what @2e3s said)

Comment (slavcodev) [reply spec/components/schemas/Subscription.yaml]: Which billing status would have the draft subscription?

**Pull request merged**